Tests/FirebaseTests/Helpers/String+Random.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+extension String {
+    static func random(length: Int) -> String {
+        let letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```

Comment:
I'm not sure if `letters` is the good name for a string that contains `0123456789`.

Reply:
Done